src/app/components/authorized/authorized.component.spec.ts

```diff
@@ -1,24 +1,53 @@
-import { HttpClientModule } from '@angular/common/http';
+import { HttpClient, HttpClientModule } from '@angular/common/http';
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 import { BrowserModule } from '@angular/platform-browser';
-import { Router } from '@angular/router';
+import { Observable, of } from 'rxjs';
+//import { Router } from '@angular/router';
 import { AppRoutingModule } from 'src/app/app-routing.module';
 import { GatewayService } from 'src/app/services/gateway.service';
 import { PostService } from 'src/app/services/post.service';
-import { SubredditService } from 'src/app/services/subreddit.service';
+import { Post } from 'src/types/posts';
+//import { SubredditService } from 'src/app/services/subreddit.service';
 import { NavigationComponent } from '../navigation/navigation.component';
 import { WelcomePageComponent } from '../welcome-page/welcome-page.component';
 import { AuthorizedComponent } from './authorized.component';
 
+class MockGatewayService {
+    mockPosts: Post[] = [
```

Comment:
I can imagine that this might be used multiple times over all the tests. It would make sense to move it to a constant in a different file if that is the case.

Reply:
I will keep this in mind. I will focus on testing the rest of the week and will do this.